.github/guides/Why This Library.md

```diff
@@ -14,7 +14,7 @@ There have been many attempts by others to create libraries that decouple the co
 Additionally, this is a concerted effort from WWT. We do a lot of software development for large companies and this problem space is something we've regularly encountered. The concept of a Workflow has been toyed with by several projects. This library has been used to power some really great experiences already, and we intend to continue using it to do even bigger and better things in the future.
 
 ### Flexibility over Assumptions
-Not everyone develops software the same way, and it would be a sad day for the industry if we thought we had all the answers. We believe strongly in a culture of innovation and we want consumers of our library to be able to make choices they feel are correct. Workflows are not an architectural pattern, they're an abstraction you can choose to use. Much like the standard libraries that come with programming languages the idea here is to give you tools that can be really useful and let you choose the best way to use them.
+Not everyone develops software the same way, and it would be a sad day for the industry if we thought we had all the answers. We believe strongly in a culture of innovation and we want consumers of our library to be able to make choices they feel are correct. Workflows are not an architectural pattern, they're an abstraction you can choose to use. Much like the standard libraries that come with programming languages, the idea here is to give you tools that can be really useful and let you choose the best way to use them.
```

Comment:
More Grammarly

```suggestion
Not everyone develops software the same way, and it would be a sad day for the industry if we thought we had all the answers. We believe strongly in a culture of innovation, and we want consumers of our library to be able to make choices they feel are correct. Workflows are not an architectural pattern; they're an abstraction you can choose to use. Much like the standard libraries that come with programming languages, the idea here is to give you tools that can be useful and let you choose the best way to use them.
```